tutorials/rise_text.ipynb

```diff
@@ -8,7 +8,7 @@
     "## Interpreting a movie review sentiment model with RISE\n",
```

Comment:
<p>I would like to add a comma after 'text' like so:</p><p>DIANNA includes a visualization package, capable of highlighting each word of a text,<strong> </strong>based on their relevance scores.</p><p>In my mind I parsed it like 'text-based' the first 2 times I read the sentence.</p><p> </p><p>Also, there is a double space in 'in  red'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>